internal/plugin/connectors/tcp/mssql/connector.go

```diff
@@ -285,7 +297,19 @@ func (connector *SingleUseConnector) waitForServerConnection(
 		// Since the communication between the client and Secretless must be unencrypted,
 		// we fool the client into thinking that it's talking to a server that does not
 		// support encryption.
```

Comment:
I believe this logic is already performed in the driver, is it not? That’s why we originally overrode the encryption here.